internal/gencli/gencli.go

```diff
@@ -208,15 +175,46 @@ func (g *gcli) buildCommands() {
 				pkg.Name,
 				cmd.InputMessageType[strings.LastIndex(cmd.InputMessageType, ".")+1:])
 
-			// build input fields into flags
-			if cmd.InputMessageType != EmptyProtoType && !cmd.ClientStreaming {
-				cmd.Flags = append(cmd.Flags, g.buildFieldFlags(&cmd, msg, "", false)...)
+			if cmd.InputMessageType != EmptyProtoType {
+				// imports handling input
+				putImport(cmd.Imports, &pbinfo.ImportSpec{
+					Name: "os",
+					Path: "os",
+				})
```

Comment:
If the `Name` is the same as the name of the package, I think we can just drop the name. Then if we `fmt.Printf("%s %q", imp.Name, imp.Path)` we get back `import "os"` as expected.